kiliautoml/models/_detectron2_semantic_segmentation.py

```diff
@@ -125,13 +126,14 @@ def train(
         batch_size: int,
         clear_dataset_cache: bool,
         disable_wandb: bool,
-        verbose: int,
         modal_train_args: ModalTrainArgs,
     ):
         """Download Kili assets, convert to coco format, then to detectron2 format, train model."""
-        _ = verbose, modal_train_args
+        _ = modal_train_args
```

Comment:
you could remove this one too if it is not too complicated

Reply:
_ = modal_train_args eans that for this modality, there is no specific arguments